app/views/pages/index.html.erb

```diff
@@ -1,5 +1,187 @@
+         Duis nunc libero, iaculis lacinia urna quis, eleifend interdum risus. Duis est turpis, vulputate sit amet odio id, finibus lacinia justo. 
+      </p>
+      <hr class="border-1 border-white rounded-md m-0">
+      <div class="flex flex-row justify-between">
+        <p>Fourth FAQ</p>
+        <div class="cursor-pointer FAQButton">
+          <span class="material-symbols-rounded">
+            arrow_drop_down_circle
+          </span>  
+        </div>    
+      </div>
+      <p class="hidden text-base font-normal tracking-wide FAQParagraph">
+         Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec at facilisis enim. Nam pretium, ante at aliquam mollis, 
+         mauris ex volutpat elit, vitae ornare urna felis a ex. Duis posuere blandit neque, et aliquam sapien. Mauris placerat 
+         accumsan leo, et posuere sem consequat in. Nam dictum vestibulum mauris sit amet gravida. Proin sagittis semper tincidunt. 
+         Duis nunc libero, iaculis lacinia urna quis, eleifend interdum risus. Duis est turpis, vulputate sit amet odio id, finibus lacinia justo. 
+      </p>
+      <hr class="border-1 border-white rounded-md m-0">
+    </section>
+		<section class="flex flex-col items-center px-[5vw] py-8 md:px-[20vw] md:py-16 text-blue-400 bg-gray-600 text-xl md:text-left">
+			<h3 class="text-center md:text-left text-blue-400 text-2xl sm:text-4xl font-semibold rounded-md md:mb-4">
+				Fill up the form with required information
+			</h3>
+			<%= form_with(model: @select_form, url:"/selection") do |f| %>
+				<div class="my-8 flex flex-col items-center">
+					<div class="text-white mr-4">
+						<%= f.label :name, placeholder: "Name" , class: "text-base"%>
+					</div>
+					<div>
+						<%= f.text_field :name , :required => 'required', class: "bg-white text-blue-500 rounded-md p-4 max-w-[50vw] md:max-w-[30vw] text-sm"  %>
+					</div>				
+				</div>
+				<div class="my-8 flex flex-col items-center">
+					<div class="text-white mr-4">
+						<%= f.label :email , class: "text-base"%>
+					</div>
+					<div>
+						<%= f.email_field :email , :required => 'required' , class: "bg-white text-blue-500 rounded-md p-4 max-w-[50vw] md:max-w-[30vw] text-sm" %>
+					</div>				
+				</div>
+				<div class="my-8 flex flex-col items-center">
+					<div class="text-white mr-4">
+						<%= f.label :select_kindergarten , class: "text-base"%>
+					</div>
+					<div class="text-blue-500 bg-white rounded-md p-4 max-w-[50vw] md:max-w-[30vw] overflow-hidden text-sm">
+						<%= f.select "kindergarten_selection", options_for_select([ "Option 1", "Option 2", "Option 3", "Option 4", "Option 5", "Name of the kindergarten, Adress" ], "Option 1"), :required => 'required' %>
+					</div>				
+				</div>
+        <div class="flex flex-row justify-center">
+          <button class="text-white bg-blue-400 hover:text-blue-400 hover:bg-white rounded-md px-5 py-3">
+					  <%= f.submit :Send %>
+				  </button>
+        </div>
```

Comment:
Please use proper indent

Reply:
For some reason even the latest commit has broken indentations. I'll try to find resolution to it.